plane/source/docs/soaring.rst

```diff
@@ -205,8 +206,20 @@ Limit maximum distance from home
    Available in firmware revisions 4.1 and later.
 
 
-If using FBWB or CRUISE mode, the parameter :ref:`SOAR_MAX_RADIUS<SOAR_MAX_RADIUS>` can be used to trigger RTL if the aircraft is more than this distance from home when thernalling in LOITER mode. Note that this parameter won't stop the aircraft from exceeding this distance before it enters LOITER mode.
+If using FBWB or CRUISE mode, the parameter :ref:`SOAR_MAX_RADIUS<SOAR_MAX_RADIUS>` can be used to trigger RTL if the aircraft is more than this distance from home when thermalling while in LOITER mode. Note that this parameter won't stop the aircraft from exceeding this distance before it enters LOITER mode.
+
+If Soaring is enabled when beyond this :ref:`SOAR_MAX_RADIUS<SOAR_MAX_RADIUS>`, it will not begin any Loitering in thermals, ie the same effect as if the middle RC switch option for Soaring was used.
```

Comment:
I think I misled you on this point, apologies. Actually it can still begin loitering but will immediately RTL. This is obviously not great behaviour and could be improved

Reply:
fixed